### PR TITLE
Database based on diagram

### PR DESCRIPTION
**Pull request by yayner2002**
in this PR, I and my partner  @ifzyy 

- created a database based on a diagram.
- the data we created is found in a file named `schema_based_on_diagram.sql`

regards

team @yayner2002  and @ifzyy

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+CREATE TABLE IF NOT EXISTS public.patients
+(
+    id integer NOT NULL GENERATED BY DEFAULT AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    name character varying COLLATE pg_catalog."default",
+    date_of_birth date,
+    CONSTRAINT patients_pkey PRIMARY KEY (id)
+)
+
+--------create treatments table ---------
+CREATE TABLE IF NOT EXISTS public.treatments
```
Comment (thecodechaser) [inline schema_based_on_diagram.sql:11]: - [ ] Please create indexes for all FOREIGN KEYS.

```ruby 
* CREATE INDEX ON medical_histories (patient_id);
* CREATE INDEX ON invoices (medical_history_id);
* CREATE INDEX ON invoice_items (invoice_id);
* CREATE INDEX ON invoice_items (treatment_id);
* CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
* CREATE INDEX ON medical_histories_has_treatments (treatment_id);

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @yayner2002, and Team👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (Whoistolu) [reply schema_based_on_diagram.sql:11]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @yayner2002 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**